examples/breakout/breakout.py

```diff
@@ -49,7 +49,6 @@
     pipeline.reset_()
     is_done = False
     while not is_done:
-        # Broken until select_softmax is implemented.
```

Comment:
Should this be implemented now?

Reply:
Yes, in this PR.

(Although the softmax action selection works, I don't think the breakout scripts actually learn anything. I didn't run enough episodes to test this.)